### DISCUSSION TITLE
Download Mp3

### DISCUSSION THREAD

**Discussion by oldvalley49** [pear-devs/pear-desktop#159] [Q&A] [answered]
Is there a way to set the location of downloaded mp3 file?

**Comment by ChickChuck2** [comment]
UP

**Comment by semvis123** [comment]
There is a way, plugins -> advanced options, see #166 for more details

**Comment by Araxeus** [comment, marked as answer]
I might try adding a Folder Chooser dialog to the options menu, if @th-ch is ok with it and isn't already doing that himself 😅 
EDIT: #207 done

**Comment by inson1** [comment]
@oldvalley49 Why dont you close this, if it is resolved? :)
